src/main/java/org/jolokia/docker/maven/assembly/DockerFileBuilder.java

```diff
@@ -264,6 +279,22 @@ public DockerFileBuilder expose(List<String> ports) {
         return this;
     }
 
+    /**
+     * Adds the RUN Commands within the build image section
+     * @param runCmds
+     * @return
+     */
+    public DockerFileBuilder runCommands(List<String> runCmds) {
```

Comment:
I would probably put runCmd != null as invariant and guard with `Objects.requireNotNull(runCmds)`.